src/main/java/cz/centrum/haffner/SimpleTrainingDavid/AppServices/MetricsInfoService.java

```diff
@@ -10,167 +13,26 @@
-                        {missingFieldsRowsCounter ++;}
-
-                    // messages with blank content
-                    if ("".equals(jsonMap.get("message_content")) ) {blankContentMessagesCounter ++;}
-
-                    // rows with field errors
-                    if ( jsonMap.get("timestamp").getClass() != Long.class ||
-                            jsonMap.get("origin").getClass() != Long.class ||
-                            jsonMap.get("destination").getClass() != Long.class ||
-                            jsonMap.get("message_content").getClass() != String.class ||
-                            !( "DELIVERED".equals(jsonMap.get("message_status")) || "SEEN".equals(jsonMap.get("message_status")) ))
-                    {fieldsErrorsRowsCounter ++;}
-
-
-
-                } else if ("".equals( jsonMap.get("message_type") )){
-                    missingFieldsRowsCounter ++;
-                } else {
-                    // field error in message_type
-                    fieldsErrorsRowsCounter ++;
-                }
-            }
-
-            // final mapping
-            groupedCallsCounter = callsCounter;  // TODO: temporary solution
-            koToOkRatio = koCallsCounter / (float) okCallsCounter;  // share of KO result to OK result
-
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-    }
-
-    public int getMissingFieldsRowsCounter() {
-        return missingFieldsRowsCounter;
-    }
-
-    public void setMissingFieldsRowsCounter(int missingFieldsRowsCounter) {
-        this.missingFieldsRowsCounter = missingFieldsRowsCounter;
-    }
 
-    public int getBlankContentMessagesCounter() {
-        return blankContentMessagesCounter;
-    }
-
-    public void setBlankContentMessagesCounter(int blankContentMessagesCounter) {
-        this.blankContentMessagesCounter = blankContentMessagesCounter;
-    }
-
-    public int getFieldsErrorsRowsCounter() {
-        return fieldsErrorsRowsCounter;
-    }
-
-    public void setFieldsErrorsRowsCounter(int fieldsErrorsRowsCounter) {
-        this.fieldsErrorsRowsCounter = fieldsErrorsRowsCounter;
-    }
-
-    public int getGroupedCallsCounter() {
-        return groupedCallsCounter;
-    }
-
-    public void setGroupedCallsCounter(int groupedCallsCounter) {
-        this.groupedCallsCounter = groupedCallsCounter;
-    }
-
-    public float getKoToOkRatio() {
-        return koToOkRatio;
-    }
-
-    public void setKoToOkRatio(float koToOkRatio) {
-        this.koToOkRatio = koToOkRatio;
-    }
-
-    public float getAverageCallDuration() {
-        return averageCallDuration;
-    }
-
-    public void setAverageCallDuration(float averageCallDuration) {
-        this.averageCallDuration = averageCallDuration;
-    }
-
-    public String getGivenWordsRanking() {
-        return givenWordsRanking;
-    }
+    public MetricsInfoDataSet proceedMetricsInfoDataSet(long requestedDate) {
```

Comment:
V praxi se častěji potkáš s použitím názvu processSomething...().
Proceed bych použil ve chvíli, kdy něco někam jen předáváš/posunuješ dál kódem - ale to je maličkost.

Reply:
ok